packaging/ubuntu-16.04/rules

```diff
@@ -17,8 +17,11 @@ export DH_GOLANG_EXCLUDES=tests
 export DH_GOLANG_GO_GENERATE=1
 
 export PATH:=${PATH}:${CURDIR}
-# make sure that correct go version is found on trusty
-export PATH:=/usr/lib/go-1.10/bin:${PATH}
+# make sure that correct go version is found on xenial
+export PATH:=/usr/lib/go-1.13/bin:${PATH}
+# go-1.13 is using $HOME to store cache files but sbuild will prevent
+# writing to the default $HOME
+export HOME:=/tmp/snapd-build-home
```

Comment:
Isn't this already controlled by the `GOCACHE` variable you set below?

Reply:
Good catch! It is not actually needed, you are right here.